lib/components/FieldLabel/FieldLabel.tsx

```diff
@@ -13,39 +13,32 @@ export interface FieldLabelProps {
   description?: ReactNode;
 }
 
-export default class FieldLabel extends Component<FieldLabelProps> {
-  static displayName = 'FieldLabel';
-
-  render() {
-    const {
-      id,
-      label,
-      secondaryLabel,
-      tertiaryLabel,
-      description,
-    } = this.props;
-
-    return label ? (
-      <Box paddingBottom="xsmall">
-        <Box component="span" display="flex" className={styles.spaceBetween}>
-          <label htmlFor={id}>
-            <Text component="span">
-              <Strong>{label}</Strong>
-              {secondaryLabel ? (
-                <Secondary>&nbsp;({secondaryLabel})</Secondary>
-              ) : null}
-            </Text>
-          </label>
-          {tertiaryLabel ? (
-            <Text component="span">&nbsp;{tertiaryLabel}</Text>
-          ) : null}
-        </Box>
-        {description ? (
-          <Box paddingTop="xxsmall" paddingBottom="xxsmall">
-            <Text color="secondary">{description}</Text>
-          </Box>
+export const FieldLabel = ({
```

Comment:
This guy doesn't house any children, so you'd be safe to `memo`, and arguably you should be on any, and every function component.